Solutions/Marain.TenantManagement.Specs/Data/ServiceManifests/ManifestWithDependencies.json

```diff
@@ -1,5 +1,9 @@
 {
   "contentType": "application/vnd.marain.tenancy.servicemanifests.servicemanifest",
+  "wellKnownTenantGuid": "31b74bd9-b45d-4402-81cf-13fc6f680a85",
   "serviceName": "Manifest with dependencies",
-  "dependsOnServiceNames": [ "Operations", "FooBar" ]
+  "dependsOnServiceTenantIds": [
+    "3633754ac4c9be44b55bfe791b1780f12429524fe7b6cc48a265a307407ec858",
```

Comment:
Hmm. I now kind of want the names in here to, mainly for readability. (But I'd also like the tooling to produce a warning it the ID supplied doesn't match the name.) The problem with switching to IDs here is that I no longer have any idea what service these things actually depend on. It's not like in general I can even search for these IDs because the manifest of the target service doesn't contain it—it contains a GUID instead.

Reply:
Did we force the well-known IDs to be GUIDs?

Reply:
Or could we make them more meaningful?

Reply:
So an encoded thing.

Reply:
Ah, but that is the encoded ID.

Reply:
A comment would be handy - as it is a fixed ID, for a fixed well-known tenant!